administrator/components/com_code/language/en-GB/en-GB.com_code.ini

```diff
@@ -5,15 +5,15 @@
 
 COM_CODE="Joomla! Code Data"
 COM_CODE_COMPONENT_CONFIG="Component Configuration"
-COM_CODE_CONFIG_EMAIL_ERROR_DESC="A comma separated list of users to e-mail when a cron job has an error."
-COM_CODE_CONFIG_EMAIL_ERROR_LABEL="E-mail Users On Error"
-COM_CODE_XML_DESCRIPTION="Component synchronizing data with JoomlaCode and other admin tasks"
+COM_CODE_CONFIG_EMAIL_ERROR_DESC="A comma separated list of users to email when a cron job has an error."
+COM_CODE_CONFIG_EMAIL_ERROR_LABEL="Email Users On Error"
+COM_CODE_XML_DESCRIPTION="Component synchronising data with JoomlaCode and other admin tasks"
 
 COM_CODE_ABOUT_TITLE="Joomla! Code Component - About"
 COM_CODE_ABOUT="About"
 
 COM_CODE_TRACKERS="Trackers"
-COM_CODE_TRACKERS_HOW_TO_EDIT="To edit the title or description of any tracker just click on it to open a inline editor"
+COM_CODE_TRACKERS_HOW_TO_EDIT="To edit the title or description of any tracker just click on it to open an inline editor"
```

Comment:
This is what happens when you have british people writing the language strings.... oh wait :P